src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -665,7 +665,7 @@ public override IOperation VisitUsingDeclaration(IUsingDeclarationOperation oper
 
         public override IOperation VisitWith(IWithOperation operation, object argument)
```

Comment:
>VisitWith [](start = 35, length = 9)

It looks like we are missing an override for INegatedPatternOperation and quite possibly for other recently added nodes. Consider either addressing that or opening an issue. Perhaps it should be possible to generate the cloner, then bugs like that could be avoided.